docs/en/defined-terms.rst

```diff
@@ -330,9 +330,13 @@ Below is the description of acronyms and definitions which are useful for furthe
       Aligned with ARF 2.5.
 
     **Wallet Unit Attestation**
+     Data object issued by a Wallet Provider that proves the keys used for key binding of Credentials reside in a trustworthy WSCD, 
+     and checks the Wallet Unit has not been revoked. Specific to IT-Wallet. 
+
+    **Wallet App Attestation**
     **Wallet Attestation**
-      Data object issued by a Wallet Provider describing the components of the Wallet Unit.
-      Aligned with ARF 2.5.
+      Data object issued by a Wallet Provider proving the integrity of the Wallet Unit.
```

Comment:
What do you think about renaming it this way? Otherwise it might be confused with WUA.
```suggestion
      Data object issued by a Wallet Provider proving the integrity of the Wallet Application.
```